scripts/covid19indiaORG/INDIA_MAP.py

```diff
@@ -0,0 +1,722 @@
+      "East Champaran":"Q49159",
+      "Gaya":"Q49173",
+      "Gopalganj":"Q49171",
+      "Jamui":"Q49168",
+      "Jehanabad":"Q49176",
+      "Kaimur":"Q77367",
+      "Katihar":"Q77568",
+      "Khagaria":"Q49175",
+      "Kishanganj":"Q77375",
+      "Lakhisarai":"Q77505",
+      "Madhepura":"Q77746",
+      "Madhubani":"Q77474",
+      "Munger":"Q77452",
+      "Muzaffarpur":"Q77731",
+      "Nalanda":"Q77633",
+      "Nawada":"Q100067",
+      "Patna":"Q100077",
+      "Purnia":"Q100082",
+      "Rohtas":"Q100085",
+      "Saharsa":"Q100120",
+      "Samastipur":"Q100117",
+      "Saran":"Q100146",
+      "Sheikhpura":"Q100093",
+      "Sheohar":"Q100095",
+      "Sitamarhi":"Q100144",
+      "Siwan":"Q100131",
+      "Supaul":"Q100139",
+      "Vaishali":"Q100130",
+      "West Champaran":"Q100124"
+   },
+   "Arunachal Pradesh":{
+      "Anjaw":"Q15413",
+      "Changlang":"Q15427",
+      "Dibang Valley":"Q15446",
+      "East Kameng":"Q15424",
+      "East Siang":"Q15419",
+      "Kamle":"Q48731073",
+      "Kra Daadi":"Q21018627",
+      "Kurung Kumey":"Q2449506",
+      "Lepa Rada":"Q63563632",
+      "Lohit":"Q15438",
+      "Longding":"Q5627568",
+      "Lower Dibang Valley":"Q2373368",
+      "Lower Siang":"Q13602925",
+      "Lower Subansiri":"Q15436",
+      "Namsai":"Q21559824",
+      "Pakke Kessang":"Q61439260",
+      "Papum Pare":"Q15432",
+      "Shi Yomi":"Q63563625",
+      "Siang":"Q18642331",
+      "Tawang":"Q15449",
+      "Tirap":"Q15448",
+      "Upper Dibang Valley":"Q15446",
+      "Upper Siang":"Q15465",
+      "Upper Subansiri":"Q15464",
+      "West Kameng":"Q15459",
+      "West Siang":"Q15453"
+   },
+   "Andhra Pradesh":{
+      "Anantapur":"Q15212",
+      "Chittoor":"Q15213",
+      "East Godavari":"Q15338",
+      "Guntur":"Q15341",
+      "Krishna":"Q15382",
+      "Kurnool":"Q15381",
+      "Prakasam":"Q15390",
+      "S.P.S. Nellore":"Q15383",
+      "Srikakulam":"Q15395",
+      "Visakhapatnam":"Q15394",
+      "Vizianagaram":"Q15392",
+      "West Godavari":"Q15404",
+      "Y.S.R. Kadapa":"Q15342"
+   }
+}
+
+STATES = {
+   "West Bengal":"Q1356",
+   "Uttarakhaland":"Q1499",
+   "Uttar Pradesh":"Q1498",
+   "Tripura":"Q13636",
+   "Telangana":"Q677037",
+   "Tamil Nadu":"Q1445",
+   "Skkim":"Q1505",
+   "Rajasthan":"Q1437",
+   "Punjab":"Q22424",
+   "Puducherry":"Q66743",
+   "Odisha":"Q22048",
+   "Nagaland":"Q1599",
+   "Mizoram":"Q1502",
+   "Meghalaya":"Q1195",
+   "Manipur":"Q1193",
+   "Maharashtra":"Q1191",
+   "Madhya Pradesh":"Q1188",
+   "Ladakh":"Q200667",
+   "Kerala":"Q1186",
+   "Karnataka":"Q1185",
+   "Jharkhand":"Q1184",
+   "Jammy and Kashmir": "Q66278313",
+   "Himachal Pradesh":"Q1177",
+   "Haryana":"Q1174",
+   "Gujarat":"Q1061",
+   "Goa":"Q1171",
+   "Delhi":"Q1353",
+   "Chhattisarh":"Q1168",
+   "Chandigarh":"Q43433",
+   "Bihar":"Q1165",
+   "Assam":"Q1164",
+   "Arunachal Pradesh":"Q1162",
+   "Andhra Pradesh":"Q1159",
+   "Andaman and Nicobar Islands":"Q40888"
+}
```

Comment:
How did you come up with this map?

Reply:
The places' dcids are resolved by wikidataId.
The map of State->District->wikidataId was generated using the following:
1. The wikidataId for each place is queried using the place_resolver.go script.
2. A script is used against wikidata.org/wiki/${wikidataId} that verifies that the place is both a District and part of India.
3. Manual check has been performed to ensure that the name matches.

Reply:
1. I see. Does the data have any better IDs?
2. Do you have that code too? Should we check it in?
3. How many places did you manually check? Ballpark is fine.

Reply:
@pradh do you have any opinions about manifesting this import to prod if we are using place name resolver to get the Wikidata IDs of Indian districts?

Reply:
@tjann I have added the script that checks against wikidataId to ensure the wikidataId is correct.
It basically goes through all the wikidataIds and exports a CSV with wikidata name and whether it belongs to India.
Then it's very easy to manually check that all are correct. I added a README.md too.

Reply:
This dataset does not have any IDs right?  So approach to mapping to wikidataId via some heuristics (including place name resolver) and manual check, sounds reasonable...